frontend/public/components/monitoring/dashboards/index.tsx

```diff
@@ -358,21 +358,26 @@ const Card: React.FC<CardProps> = ({ panel }) => {
 
   const panelClassModifier = getPanelClassModifier(panel);
   return (
-    <div
-      className={`monitoring-dashboards__panel monitoring-dashboards__panel--${panelClassModifier}`}
-    >
-      <DashboardCard
-        className="monitoring-dashboards__card"
-        gradient={panel.type === 'grafana-piechart-panel'}
+    (panel.type === 'grafana-piechart-panel' ||
+      panel.type === 'graph' ||
+      panel.type === 'singlestat' ||
+      panel.type === 'table') && (
```

Comment:
I think using an early return would be a little more readable. Also, since `panel.type` is repeated 4 times, maybe use `Array.includes`?

```
if (!['grafana-piechart-panel', 'graph', 'singlestat', 'table'].includes(panel.type)) {
  return null;
}
```

Reply:
All set!